.github/workflows/postgres.yml

```
@@ -0,0 +1,65 @@
+  workflow_dispatch:
+
+jobs:
+  publish-docker-images:
+    name: "publish docker images"
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
```

actionlint

the runner of "docker/setup-buildx-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)